src/k8s-extension/azext_k8s_extension/partner_extensions/azuremonitormetrics/dc/defaults.py

```diff
@@ -13,10 +13,11 @@
 # All DC* object names should end only in alpha numeric (after `length` trim)
 # DCE remove underscore from cluster name
 def sanitize_name(name, objtype, length):
+    name = name[0:length]
     length = length - 1
     if objtype == DC_TYPE.DCE:
         name = name.replace("_", "")
```

Comment:
I don't believe there is a need for this line now

Reply:
removed